core/settings.py

```
@@ -0,0 +1,654 @@
+from loguru import logger
+
+
+def log_warning(message:str, category, filename:str, lineno:str, file=None, line=None) ->  None:
+     logger.warning(f"{filename}:{lineno} - {category.__name__}: {message}")  
+
+TEMP_STATIC_DIR = tempfile.mkdtemp()
+os.environ['TEMP_STATIC_DIR'] = TEMP_STATIC_DIR
+NSFW_WORD_LIST_URI = values.URLValue(
+    environ=True, environ_prefix=None, environ_name="NSFW_WORD_LIST_URI"
+)
+GLOBAL_NOW = datetime.now()
```

Ruff (0.12.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)